man/check-os-release-simple.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/python
```

Comment:
env python ?

Reply:
It's visible in the man page, so we don't use `env`. It's the same in the other example.